setup.py

```diff
@@ -24,6 +24,7 @@
     'pytest-flakes>=1.0.1',
     'pytest-pep8>=1.0.6',
     'pytest>=3.0.5',
+    'sqlalchemy>=1.1.6',
```

Comment:
I don't find a proper way to test the minimal version, so I do it by hand. If you could give some tip about this it would be awesome.